xbmc/addons/kodi-dev-kit/include/kodi/c-api/addon-instance/inputstream.h

```diff
@@ -179,6 +179,9 @@ extern "C"
 
     /// @brief **5 :** To identify @ref cpp_kodi_addon_inputstream_Defs_Info as Radio RDS
     INPUTSTREAM_TYPE_RDS,
+
+    /// @brief **6 :** To identify @ref cpp_kodi_addon_inputstream_Defs_Info as Radio ID3
+    INPUTSTREAM_TYPE_ID3,
```

Comment:
Latest force-pushed commit does not contain the api version bump anymore. As you're appending a new enum value here, inputstream api version (but not min version) needs to be bumped.

Reply:
It would work for TV if the addon did its own demuxing (do any do that anymore), and it was an audio-only channel. I know these exist in the U.K., so it's likely they also exist in Europe.  What if we renamed it to "AudioID3" instead of "RadioID3"?

Will re-add the version bump to InputStream.